docs/mocksupport.rst

```diff
@@ -242,13 +242,13 @@ if there are classes in the repository.
   setting key properties as a dynamic provider for the class in a real WBEM
   server might have. This operation requires that the mock repository is in
   full mode and that the class of the new instance exists in the mock
-  repository. It fails with not supported if the mock repository is in lite
+  repository. It fails with NOT_SUPPORTED if the mock repository is in lite
```

Comment:
Change "NOT_SUPPORTED" to the full name of the CIM status code "CIM_ERR_NOT_SUPPORTED".

Reply:
DONE